src/Acquisition/tests.py

```diff
@@ -3685,12 +3453,21 @@ def test_suite():
         unittest.makeSuite(TestCreatingWrappers),
         unittest.makeSuite(TestPickle),
         unittest.makeSuite(TestInterfaces),
+        unittest.makeSuite(TestMixin),
```

Comment:
May I suggest removing all of the explicit calls to `makeSuite(SomeTestClass)` and replacing them with a single `unittest.defaultTestLoader.loadTestsFromName(__name__)`? That makes adding new test classes less error prone.

Reply:
Excellent idea, added in second commit